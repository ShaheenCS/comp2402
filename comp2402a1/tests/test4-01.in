apple
actor
amber
ball
big
bone
cat
cool
car
daddy
donkey
door
example
elephant
enter
for
fool
fight
gay
great
gamble
hockey
hall
height
ink
india
interest
jockey
jolly
job
kick
kind
kill
lemon
leprechaun
leach
mom
mall
might
negative
night
nobody
quart
quantity
quote
raid
roof
rowdy
safe
sell
sex
titan
telephone
time
urban
up
under
victory
video
violence
war
want
will
xylophone
xbox
xerox
youtube
young
yacht
zoo
zesty
zap