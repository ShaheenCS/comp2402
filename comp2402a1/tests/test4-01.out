sell
safe
rowdy
roof
raid
quote
quantity
quart
nobody
night
negative
might
mall
mom
leach
leprechaun
lemon
kill
kind
kick
job
jolly
jockey
interest
india
ink
height
hall
hockey
gamble
great
gay
fight
fool
for
enter
elephant
example
door
donkey
daddy
car
cool
cat
bone
big
ball
amber
actor
apple
zap
zesty
zoo
yacht
young
youtube
xerox
xbox
xylophone
will
want
war
violence
video
victory
under
up
urban
time
telephone
titan
sex
